iwyu_path_util.h

```diff
@@ -69,14 +69,22 @@ string MakeAbsolutePath(const string& base_path, const string& relative_path);
 // Get the parent of path.
 string GetParentPath(const string& path);
 
+// Tries to strip the prefix_path from the front of path
+// path assumed to be normalized but either absolute or relative
+// Returns true, if path was stripped
+bool StripPathPrefix(string* path, const string& prefix_path);
+
 // Below, we talk 'quoted' includes.  A quoted include is something
 // that would be written on an #include line, complete with the <> or
 // "".  In the line '#include <time.h>', "<time.h>" is the quoted
 // include.
 
 // Converts a file-path, such as /usr/include/stdio.h, to a
 // quoted include, such as <stdio.h>.
-string ConvertToQuotedInclude(const string& filepath);
+string ConvertToQuotedInclude(const string& filepath, const string& includer_path = "");
+
+// Removes quotes ("" or <>) from a string
+string StripQuotes(string s);
```

Comment:
This would be more consistent with the other Strip functions if it took a `string*` and returned void.